### DISCUSSION TITLE
What is the NFQUEUE in iptables ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#606] [General]
Hi everyone,

I noticed a new rules in my iptables generated by opensnitch

```
Chain INPUT (policy ACCEPT)
target     prot opt source               destination         
NFQUEUE    udp  --  anywhere             anywhere             udp spt:domain NFQUEUE num 0 bypass

Chain FORWARD (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination 
```

What does that rules ? (sorry no knowledge yet about iptables..)

Thanks.

**Comment by gustavo-iniguez-goya** [comment]
NFQUEUE is the netfilter target that allow us to intercept and block or allow a connection:

https://www.netfilter.org/projects/libnetfilter_queue/
https://home.regit.org/netfilter-en/using-nfqueue-and-libnetfilter_queue/
